srai/loaders/osm_way_loader/osm_way_loader.py

```diff
@@ -371,3 +380,23 @@ def _to_wide(self, gdf: gpd.GeoDataFrame, gdf_exploded: gpd.GeoDataFrame) -> gpd
         )
 
         return gdf_edges_wide
+
+    def _unify_index_and_columns_names(self, gdf_edges: gpd.GeoDataFrame) -> gpd.GeoDataFrame:
+        """
+        Make naming of index and columns consistent.
+
+        Args:
+            gdf_edges (gpd.GeoDataFrame): Edges to unify
+
+        Returns:
+            gpd.GeoDataFrame: Edges with unified index and columns names
+        """
+        gdf = gdf_edges.reset_index().drop(columns=["u", "v"])
+        gdf.index.name = FEATURES_INDEX
```

Comment:
```suggestion
        gdf.index.rename(FEATURES_INDEX, inplace=True)
```
It's a more modern version using the new (>0.13) Pandas API.